pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computetargethttpsproxy/crossregiontargethttpsproxy/create.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeTargetHTTPSProxy
```

Comment:
Thank you for adding the test!

Note this is what we call "integration test" and is not going to show up in the "samples" of resource reference page:
https://cloud.google.com/config-connector/docs/reference/resource-docs/compute/computetargethttpsproxy#sample_yamls

Optionally, if you want to demonstrate how this new field can be used in the resource reference page, you will need to add a new sample in the folder below.
 
https://github.com/GoogleCloudPlatform/k8s-config-connector/tree/master/config/samples/resources/computetargethttpsproxy

Reply:
Thanks for the suggestion! A sample test is added.